R/input-numeric.R

```diff
@@ -30,13 +30,14 @@
 #'
 #' @export
 numericInput <- function(inputId, label, value, min = NA, max = NA, step = NA,
-  width = NULL) {
+  width = NULL, ..., updateOn = c("input", "blur")) {
```

Comment:
```suggestion
  width = NULL, ..., updateOn = c("change", "blur")) {
```

Instead of `"input"`, would it make sense to call this `"change"`? From a UX perspective, I think most people would read `updateOn = "change"` as _updates on changes_, which is correct. But having an input that updates on input is confusing.

Reply:
Yes, I think`"change"` makes sense.

Reply:
Oh, I remember why I called it `input` instead of `change`. In the JS event listeners, there is an `input` event, which is the one that this parameter actually affects, and a different `change` event which this parameter does not affect.

Still, despite the mismatch in naming, I think your suggestion makes sense to make it easier for users to understand.